README.md

```diff
@@ -187,6 +187,22 @@ class User < ActiveRecord::Base
 end
 ```
 
+- **Modify the token lifetime**
```

Comment:
Actually the instructions here should be to change knock.rb config file to define the proper lifetime:

```
# How long before a token is expired. If nil is provided,
# token will last forever.
mattr_accessor :token_lifetime
self.token_lifetime = 1.day
```
or
```
# How long before a token is expired. If nil is provided,
# token will last forever.
mattr_accessor :token_lifetime
self.token_lifetime = {
  user: 1.day,
  admin: 30.minutes
}
```